keyboards/gmmk/pro/rev1/ansi/keymaps/apazzy/keymap.c

```diff
@@ -0,0 +1,171 @@
+};
+
+bool mouse_jiggle_mode = false; // Mouse Jiggle variable
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [WIN_BASE] = LAYOUT(
+        KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_PSCR,          KC_MJCX,
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_DEL,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,          KC_PGUP,
+        MO(WIN_FN),   KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,           KC_PGDN,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT, KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(WIN_FN),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+
+    [WIN_FN] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_MJST,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______, _______, _______, KC_PGUP, KC_UP,   KC_PGDN, KC_PSCR, _______, _______, KC_RST2,          _______,
+        _______, _______, _______, _______, _______, _______, KC_HOME, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,          _______,          _______,
+        _______,          _______, _______, _______, _______, _______, KC_END,  NK_TOGG, _______, _______, _______,          _______, _______, KC_HOME,
+        _______,KC_WINLCK,_______,                            _______,                            _______, _______, _______, _______, _______, _______
+    ),
+
+};
+
+#ifdef ENCODER_MAP_ENABLE
+    const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+        [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+        [1] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) }
+    };
+#endif
+
+void matrix_scan_user(void) {
+    if (mouse_jiggle_mode) {
+        tap_code(KC_MS_UP);
+        tap_code(KC_MS_DOWN);
+        tap_code(KC_MS_LEFT);
+        tap_code(KC_MS_RIGHT);
+        tap_code(KC_MS_WH_UP);
+        tap_code(KC_MS_WH_DOWN);
+    } else {
+    }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_RST2:
+            if (record->event.pressed) {
+                reset_keyboard();
+            }
+            return false;
+		// GUI lock toggle
+        case KC_WINLCK:
+            if (record->event.pressed) {
+                keymap_config.no_gui = !keymap_config.no_gui; //toggle status
+            }
+            return false;
```

Comment:
The [magic](https://docs.qmk.fm/#/keycodes_magic)  keycode `GU_TOGG` should be used instead.
```suggestion
```